repo_id,repo_full_name,file,file_sha1,match
380672428,raul23/pyyaml-CVE-2020-14343,README.rst,063cf4c0172208338d8fb22abffe18c452684914,CVE-2020-14343
380672428,raul23/pyyaml-CVE-2020-14343,README.rst,063cf4c0172208338d8fb22abffe18c452684914,CVE-2020-1747
380672428,raul23/pyyaml-CVE-2020-14343,test_yaml_vuln/server.py,67ccd398979e33959bf15194fc6dd479defef96d,CVE-2020-14343
380672428,raul23/pyyaml-CVE-2020-14343,test_yaml_vuln/client.py,ba11c2e2190923a44661ab0d0dbfbf0b2e4aa2e9,CVE-2020-14343
380672428,raul23/pyyaml-CVE-2020-14343,setup.py,e0f163161428a776a1fe110c3068d05547eceb7c,CVE-2020-14343
380672428,raul23/pyyaml-CVE-2020-14343,_GITHUB_REPO_METADATA_,,CVE-2020-14343
